Tools/scripts/serve.py

```diff
@@ -28,7 +28,7 @@ def app(environ, respond):
     path = sys.argv[1]
     port = int(sys.argv[2]) if len(sys.argv) > 2 else 8000
     httpd = simple_server.make_server('', port, app)
-    print("Serving {} on port {}, control-C to stop".format(path, port))
+    print("Serving {0} on 0.0.0.0 port {1} (http://0.0.0.0:{1}/), control-C to stop".format(path, port))
```

Comment:
I don't think (I may have miseed something) querying to `0.0.0.0` is well specified (while binding to to `0.0.0.0` is), I'd prefer to see 127.0.0.1 here.

Reply:
ok but I should to limit to the loopback interface

Reply:
Why limiting to listening to localhost? You're changing the behavior here.